artifacts/flagger/crd.yaml

```diff
@@ -198,6 +198,9 @@ spec:
                     portDiscovery:
                       description: Enable port dicovery
                       type: boolean
+                    headless:
```

Comment:
Please add this as `headless: false` to the docs here https://docs.flagger.app/usage/how-it-works#canary-service